src/header/header.scss

```diff
@@ -266,3 +271,70 @@
     }
   }
 }
+
+.iui-split-menu-header-button {
+  display: flex;
+  height: 100%;
+  padding: 0 !important;
+
+  // Sizes
+  > .iui-button {
+    padding: $iui-xs;
+  }
+
+  > .iui-small {
+    padding: $iui-xxs;
+  }
+
+  > .iui-large {
+    padding: $iui-s;
+  }
+
+  .iui-button:first-child {
+    padding-right: 0 !important;
+  }
+
+  .iui-split-menu {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    padding-left: 0 !important;
+    padding-right: $iui-xs !important;
+
+    &.iui-active {
+      svg {
+        fill: var(--iui-icons-color-primary);
+      }
+    }
+
+    svg {
+      border-radius: 50%;
+      padding: $iui-xs;
+    }
```

Comment:
Why does the circular shape get cutoff at the top & bottom within the slim header?
![Screen Shot 2021-07-28 at 5 19 01 PM](https://user-images.githubusercontent.com/849817/127397773-b7e913fd-a410-4ae3-9688-bf2bb08683f5.png)

Reply:
It needs a different radius for slim size

Reply:
I think this fixed itself when css class was moved.

Reply:
Can confirm, I'm not seeing the issue anymore.